subsystems/hexe_edm/data/slowcontrol.json

```diff
@@ -185,16 +221,21 @@
         return function(e, o) {
         if (e !== null) {
           return;}
-        $('.' + x.id).val(x.v);
+        if (o.rows.length != 1) {
+          alert(x.value + " not found");
+          return;
+        }
+        var v = o.rows[0].value;
+        $('.' + x.name).val(v);
         last_sync = new Date();
         };
     };
 
```

Comment:
call get_database with the correct db:

either:

`nedm.get_database()`

or 

`nedm.get_database('nedm%2Fhexe_edm')`